### DISCUSSION TITLE
Frequently Asked Questions (FAQ)

### DISCUSSION THREAD

**Discussion by hamza-usmani** [microsoft/WSA#51] [Announcements]
# Frequently Asked Questions

Windows Subsystem for Android™ and the Amazon Appstore will no longer be available in the Microsoft Store after March 5, 2025. [Learn more](https://learn.microsoft.com/en-us/windows/android/wsa/).

## General

How do I start using the Windows Subsystem for Android™️?
> [Install the Amazon Appstore](https://support.microsoft.com/windows/install-mobile-apps-and-the-amazon-appstore-on-windows-f8d0abb5-44ad-47d8-b9fb-ad6b1459ff6c) to get started!

What are the minimum requirements to install the subsystem?
> Your device must meet minimum Windows 11 requirements, as well as feature-specific requirements: [Device requirements](https://support.microsoft.com/en-us/windows/install-mobile-apps-and-the-amazon-appstore-on-windows-f8d0abb5-44ad-47d8-b9fb-ad6b1459ff6c).

Why does the subsystem require enabling Virtual Machine Platform?
> Virtualization lets your PC emulate a different operating system, like Android™ or Linux. Enabling virtualization gives you access to a larger library of apps to use and install on your PC. [Learn more about virtualization](https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1).

Does the subsystem support Google Play Services?
> Windows Subsystem for Android is based on [Android Open Source Project](https://source.android.com), which does not include proprietary Google Play Services.  Google governs the licenses for Google Play Services.

## Installing the subsystem

How do I download the subsystem, and what does the package consist of?
> When you [download the Amazon Appstore](http://aka.ms/AmazonAppstore) from the Microsoft Store, you’ll also be downloading the Windows Subsystem for Android. Inside that package you’ll find both the Linux kernel and the Android OS.

Why is the subsystem a separate application?
> We’ve intentionally made the subsystem a Microsoft store application (just like we did for Windows Subsystem for Linux) so that we can more quickly/easily update the project to fix issues related to application compatibility and to keep up with monthly Linux and Android security updates.

What version of the Android OS is shipping in the subsystem?
> The Android OS we’re currently shipping is based on the Android Open Source Project (AOSP) version AOSP 12.1. We made every effort to use AOSP as-is, and we only needed to make a small handful of targeted changes to the OS.

## Subsystem settings

What is the Windows Subsystem for Android Settings app and how do I access it?
> The Windows Subsystem for Android Settings app can be used to adjust a number of subsystem settings, access files, enable developer mode and much more. To access the app, select Start, search for Windows Subsystem for Android Settings, and open it. [Learn more about the Settings app](https://support.microsoft.com/en-us/windows/manage-settings-for-mobile-apps-on-windows-000f97e8-8c20-490e-9ef4-cd90d903f847#:~:text=If%20you%20need%20to%20adjust,a%20trademark%20of%20Google%20LLC.).

_Android is a trademark of Google LLC._
_Amazon and all related marks are trademarks of Amazon.com, Inc., or its affiliates._

**Comment by dhirensham** [comment]
@hamza-usmani can you please share plans to expand availability beyond the current limited set of countries that are supported?
